src/Google.Maps/StaticMaps/StaticMapService.cs

```diff
@@ -77,11 +78,9 @@ public int GetMapToStream(StaticMapRequest mapOptions, System.IO.Stream outputSt
 
 			int totalBytes = 0;
 
-			WebRequest request = WebRequest.Create(requestUri);
-
-			using(WebResponse response = request.GetResponse())
+			using(HttpClient client = new HttpClient())
 			{
-				Stream inputStream = response.GetResponseStream();
+                Stream inputStream = client.GetStreamAsync(requestUri).Result;

```

Comment:
You're probably right, I don't have time to check on it right now.  There was a pull request for async versions of the methods but the guy was too unfocused and made a lot of other changes.  

We'll implement async/await methodology after this pull request.  Tricky part is getting it all in there in limited time allotments  :)